docs/dev/new_check_howto.md

```diff
@@ -407,6 +407,10 @@ The complete list of mandatory and optional attributes for the `manifest.json` f
 | `metric_to_check`    | String          | Optional           | The presence of this metric determines if this integration is working properly. If this metric is not being reported when this integration is installed, the integration is marked as broken in the Datadog application. |
 | `metric_prefix`      | String          | Optional           | The namespace for this integration's metrics. Every metric reported by this integration will be prepended with this value.                                                                                               |
 | `process_signatures` | Array of String | Optional           | A list of signatures that matches the command line of this integration.                                                                                                                                                  |
+| `assets`       | Array | Mandatory          | Relative location of where certain asset files live and their name.                                                                                                                                             |
```

Comment:
All those are not arrays but dictionaries, right?

Reply:
Correct, updated